src/mlpack/methods/ann/layer/elu.hpp

```diff
@@ -206,27 +210,29 @@ class ELU
    * Computes the first derivative of the activation function.
    *
    * @param x Input data.
+   * @param y Propagated data.
```

Comment:
Can you make this comment just a little more clear, in that the propagated data is actually equal to `f(x)`?